README.md

```diff
@@ -6,23 +6,28 @@
 ![Tensorflow](https://img.shields.io/badge/tensorflow-v2.9.0+-success.svg)
 [![Contributions Welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/keras-team/keras-cv/issues)
 
-KerasCV is a library of modular computer vision oriented Keras components.
-These components include models, layers, metrics, losses, callbacks, and utility
-functions.
+KerasCV is a library of modular, framework-agnostic computer vision components 
```

Comment:
Should we sell more here? IMO this is kind of a killer feature, you can write your code once, run it on multiple backends right away. Not sure exactly what the right wording is, but I think we could try to underline here a bit more.

Reply:
I think making the first sentence a bit more brief has more punch:

KerasCV is a library of modular compute vision components that work natively with TensorFlow, JAX, or PyTorch